tests/phpunit/tests/formatting/translateSmiley.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @group formatting
+ * @group emoji
+ *
+ * @covers ::translate_smiley
+ */
+class Tests_Formatting_TranslateSmiley extends WP_UnitTestCase {
+	/**
+	 * @ticket 54827
+	 */
+	public function test_translate_smiley_should_return_empty_string() {
+		$this->assertSame( '', translate_smiley( ':)' ) );
```

Comment:
Suggestion: also add a test for `null`, which is what started this PR. 😉

Reply:
Whoa this one went off my radar >.<

Right you are, I've updated the PR to include a test for `null`. If you get time to review the latest changes, that'd be great! :)